vet.sh

```diff
@@ -119,7 +119,7 @@ fi
 SC_OUT="$(mktemp)"
 staticcheck -go 1.9 -checks 'inherit,-ST1015' ./... > "${SC_OUT}" || true
 # Error if anything other than deprecation warnings are printed.
-(! grep -v "is deprecated:.*SA1019" "${SC_OUT}")
+(! grep -Pv "\((SA1019)|(SA6002)\)$" "${SC_OUT}")
```

Comment:
What's 6002?

We can ignore this, if it's appropriate to, by changing the command above to include "-SA6002" instead.

Reply:
SA6002 complains about the use of `sync.Pool` with a non-struct type (i.e. `[]byte`). It wants us to use a `bytes.Buffer`, but we can't change that now...

Reply:
Also done.